extension/test/newman/collection.json

```diff
@@ -87,10 +87,188 @@
+											"pm.test('Cart item with input text is not there', () => {",
+											"    pm.expect(desiredCartItem).to.be.an('undefined')",
+											"})",
+											"const desiredCartItem2 = response.cartItems.find(item => item.product && item.product.id === '370')",
+											"pm.test('Cart item with input text is not there', () => {",
+											"    pm.expect(desiredCartItem2).to.be.an('undefined')",
+											"})",
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{}"
+								},
+								"url": {
+									"raw": "{{domain}}{{endpoint_cart_get}}",
+									"host": [
+										"{{domain}}{{endpoint_cart_get}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Add a simple product with text field input",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "5cd67dd2-faf6-4720-8031-cbf3a65d0a32",
+										"exec": [
+											"const response = pm.response.json()",
+											"",
+											"pm.test('Status code is 200', function () {",
+											"    pm.response.to.have.status(200);",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"products\": [\n\t\t{\n\t\t\t\"productId\": 553,\n\t\t\t\"quantity\": 1,\n\t\t\t\"properties\": [\n\t\t\t\t{\n\t\t\t\t\t\"type\": \"input\",\n\t\t\t\t\t\"id\": \"7\",\n\t\t\t\t\t\"value\": \"DEMO Text\"\n\t\t\t\t}\n\t\t\t\t]\n\t\t}\n\t\t]\n}"
+								},
+								"url": {
+									"raw": "{{domain}}{{endpoint_cart_products_add}}",
+									"host": [
+										"{{domain}}{{endpoint_cart_products_add}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Product with input text field exists",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "5cd67dd2-faf6-4720-8031-cbf3a65d0a32",
+										"exec": [
+											"const response = pm.response.json()",
+											"",
+											"pm.test('Status code is 200', function () {",
+											"    pm.response.to.have.status(200);",
+											"});",
+											"",
+											"const desiredCartItem = response.cartItems.find(item => item.product && item.product.id === '553')",
+											"pm.test('Existing cart item with input text', () => {",
+											"    const {product: {properties}} = desiredCartItem",
+											"    pm.expect(desiredCartItem).to.be.an('object')",
```

Comment:
this needs to move above the usage `const {product: {properties}} = desiredCartItem"`